astropy/time/core.py

```diff
@@ -2460,6 +2488,32 @@ def _make_value_equivalent(self, item, value):
                                  'object: {}'.format(err))
         return value
 
+    def isclose(self, other, atol=None):
+        """Returns a boolean or boolean array where two TimeDelta objects are
+        element-wise equal within a time tolerance.
+
+        Parameters
+        ----------
+        other : `~astropy.units.Quantity` or `~astropy.time.TimeDelta`
+            Quantity or TimeDelta object for comparison.
+        atol : `~astropy.units.Quantity` or `~astropy.time.TimeDelta`
+            Tolerance for equality with units of time (e.g. ``u.s`` or ``u.day``).
+            Default is one bit in the 128-bit JD time representation, equivalent
+            to about 20 picosecs.
+        """
```

Comment:
Possibly contrary to my argument above, `TimeDelta` perhaps should have a different implementation since `rtol` does make sense for `TimeDelta`.